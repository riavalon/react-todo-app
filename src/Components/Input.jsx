import React from 'react'
import styles from '../Styles/Input.module.css'

const Input = props => {
  return (
    <input
      type="text"
      placeholder={props.placeholder}
      value={props.value}
      onChange={props.onChange}
      className={styles.input}
    />
  )
}
export default Input