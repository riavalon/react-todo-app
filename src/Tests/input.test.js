import React from 'react';
import renderer from 'react-test-renderer';
import Input from '../Components/Input';

it('renders correctly', () => {
  const tree = renderer.create(
    <Input 
      placeholder="Start typing here..." 
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});