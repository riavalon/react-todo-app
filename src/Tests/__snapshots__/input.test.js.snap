// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<input
  className="input"
  placeholder="Start typing here..."
  type="text"
/>
`;
